### DISCUSSION TITLE
Works perfectly with Nepali Telecom Opearators Ncell and NTC tested on Pixel 6 global version

### DISCUSSION THREAD

**Discussion by suraj84p** [kyujin-cho/pixel-volte-patch#99] [Carrier-specific Experiences]
Works perfectly with both Nepali Telecom Opearators Ncell and NTC tested on Pixel 6 global version

**Comment by eranlot** [comment]
Are you talking about VOLTE, VOWIFI, or both?

**Reply by suraj84p** [reply]
VOLTE only... VOWIFI is currently not available in Nepal

**Comment by kyujin-cho** [comment]
Hi! Thanks for the report. I've updated the compatibility matrix page with marking NTC and Ncell as supported.

**Comment by suraj84p** [comment]
VoWifi for Ncell is working perfectly...
